Utils/patchArticles.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios'
+
+const NCnewsAPI = axios.create({
+  baseURL: "https://nc-news-tm65.onrender.com",
+});
+
+export const patchArticle = (id) => {
+return NCnewsAPI.patch(`/api/articles/${id}`)
```

Comment:
There is no object being passed to the patch request here so this is not making a sucessful request to the api.